index.html

```diff
@@ -162,32 +177,51 @@ <h1 class="animated fadeInDown delay-07s">Mihov Development</h1>
         <div class="container">
             <h2>Portfolio</h2>
             <h6>Our completed projects</h6>
-            <div class="portfolioFilter">
+            <!--<div class="portfolioFilter">
                 <ul class="Portfolio-nav wow fadeIn delay-02s">
                     <li><a href="#" data-filter="*" class="current">All</a></li>
                     <li><a href="#" data-filter=".branding">Branding</a></li>
                     <li><a href="#" data-filter=".webdesign">Web design</a></li>
                     <li><a href="#" data-filter=".printdesign">Print design</a></li>
                     <li><a href="#" data-filter=".photography">Photography</a></li>
                 </ul>
-            </div>
+            </div>-->
         </div>
         <div class="portfolioContainer wow fadeInUp delay-04s">
             <div class=" Portfolio-box printdesign">
-                <a href="#"><img src="img/Portfolio-pic1.jpg" alt=""></a>
+                <div class="Portfolio-box-content">
+                    <a href="#"><img src="img/Portfolio-pic1.jpg" alt=""></a>
+                    <div class="overlay">The content to be displayed on :hover
+                    </div>
+                </div>
+                <br>
+                <br>
                 <h3>Foto Album</h3>
                 <p>Print Design</p>
             </div>
             <div class="Portfolio-box webdesign">
-                <a href="#"><img src="img/Portfolio-pic2.jpg" alt=""></a>
+                <div class="Portfolio-box-content">
+                    <a href="#"><img src="img/Portfolio-pic2.jpg" alt=""></a>
+                    <div class="overlay">The content to be displayed on :hover
+                    </div>
+                </div>
+                <br>
+                <br>
                 <h3>Luca Theme</h3>
                 <p>Web Design</p>
             </div>
             <div class=" Portfolio-box branding">
-                <a href="#"><img src="img/Portfolio-pic3.jpg" alt=""></a>
+                <div class="Portfolio-box-content">
+                    <a href="#"><img src="img/Portfolio-pic3.jpg" alt=""></a>
+                    <div class="overlay">The content to be displayed on :hover
+                    </div>
+                </div>
+                <br>
+                <br>
                 <h3>Uni Sans</h3>
                 <p>Branding</p>
             </div>
+            <!--
```

Comment:
Please remove dead code; If we keep some for reusing purposes, then only leave one snippet, not all of them